collect_app/src/androidTest/assets/all-widgets.xml

```diff
@@ -0,0 +1,16 @@
ype="date"/><bind nodeset="/all-widgets/date_time_widgets/coptic_date_widget" type="date"/><bind nodeset="/all-widgets/date_time_widgets/islamic_date_widget" type="date"/><bind nodeset="/all-widgets/geopoint_widgets/geopoint_widget" type="geopoint"/><bind nodeset="/all-widgets/geopoint_widgets/geopoint_widget_placementmap" type="geopoint"/><bind nodeset="/all-widgets/geopoint_widgets/geopoint_widget_maps" type="geopoint"/><bind nodeset="/all-widgets/geopoint_widgets/geotrace_widget" type="geotrace"/><bind nodeset="/all-widgets/geopoint_widgets/geoshape_widget" type="geoshape"/><bind nodeset="/all-widgets/geopoint_widgets/osm_road" type="binary"/><bind nodeset="/all-widgets/geopoint_widgets/osm_building" type="binary"/><bind nodeset="/all-widgets/select_one_widgets/select_one_widget" type="select1"/><bind nodeset="/all-widgets/select_one_widgets/spinner_widget" type="select1"/><bind nodeset="/all-widgets/select_one_widgets/select_one_autoadvance_widget" type="select1"/><bind nodeset="/all-widgets/select_one_widgets/select_one_search_widget" type="select1"/><bind nodeset="/all-widgets/select_one_widgets/select_one_autocomplete_widget" type="select1"/><bind nodeset="/all-widgets/select_one_widgets/grid_widget" type="select1"/><bind nodeset="/all-widgets/select_one_widgets/grid_widget_compact" type="select1"/><bind nodeset="/all-widgets/select_one_widgets/grid_widget_compact2" type="select1"/><bind nodeset="/all-widgets/select_one_widgets/grid_widget_quickcompact" type="select1"/><bind nodeset="/all-widgets/select_one_widgets/grid_widget_quickcompact2" type="select1"/><bind nodeset="/all-widgets/select_one_widgets/select_one_image_map" type="select1"/><bind nodeset="/all-widgets/select_multi_widgets/select_multi_widget" type="select"/><bind nodeset="/all-widgets/select_multi_widgets/grid_multi_widget_compact" type="select"/><bind nodeset="/all-widgets/select_multi_widgets/grid_multi_widget_compact2" type="select"/><bind nodeset="/all-widgets/select_multi_widgets/spinner_multi_widget" type="select"/><bind nodeset="/all-widgets/select_multi_widgets/select_multi_image_map" type="select"/><bind nodeset="/all-widgets/table_list_test/table_list_test_label" type="select1"/><bind nodeset="/all-widgets/table_list_test/table_list_test_label_2" type="select"/><bind nodeset="/all-widgets/table_list_test/table_list_1" type="select1"/><bind nodeset="/all-widgets/table_list_test/table_list_2" type="select"/><bind nodeset="/all-widgets/table_list_test/list_widget" type="select1"/><bind nodeset="/all-widgets/table_list_test/list_multi_widget" type="select"/><bind calculate="concat('uuid:', uuid())" nodeset="/all-widgets/meta/instanceID" readonly="true()" type="string"/></model></h:head><h:body><input ref="/all-widgets/intro"><label>Welcome to ODK Collect! This form showcases the different available question types (_widgets_).</label><hint>This is a sample for introductory and debugging purposes. It is updated periodically so check "Get Blank Form" to get the latest! 
```

Comment:
It looks like this only has the introduction and that the actual form is missing! 

Also, why did you replace `all_widgets.xml` with `all-widgets.xml`? Is it because then it doesn't need to be renamed to be copied over after conversion? If so, that seems like a good change. That kind of explanation is a good thing to include in a commit comment (e.g. `git commit -m "Move all_widgets.xml to all-widgets.xml" -m "To match the XLSForm filename.`

Reply:
Yep, I should start working on improving my commit messages. I'll check for the actual form as well.